##### System Parameters #####
SYS:
  # Default: 20; 
  # Note: frequency for backend processing(MSCKF update), increase if more data need for processing ( keep it as camera fequency)
  backend_hz: 15

  # Default: 1; (0: static; 1:DVL-aided; 2:SFM-aided)
  # Note: Estimate IMU bias and find out the rotation matrix between inertial world frame and IMU body frame;
  imu_init_mode: 1      

  # Default: 10
  # Note: how many IMU data is selected to detect IMU jump (suddenly move)  
  imu_windows: 10

  # Default: 0.2
  # Note: the IMU variance threshold that indicates IMU jump inclued
  imu_var: 0.2

  # Default: 0.07
  # Note: IMU accleraion difference in x-axis(forward direction) shows at the suddenly movement time
  imu_delta: 0.07

  # Default: 5
  # Note: how many DVL data is selected to detect DVL jump (suddenly move)  
  dvl_windows: 4
  
  # Default: 0.05
  # Note: DVL velocity difference in x-axis(forward direction) shows at the suddenly movement time
  dvl_delta: 0.05


#### MSCKF parameters
MSCKF:
  # enable DVL exterisic rotation calibration 
  dvl_exterisic_R: true
  # enable DVL exterisic translation calibration 
  dvl_exterisic_p: true
  # enable DVL time offset calibration
  dvl_timeoffset: true
  # enable DVL scale calibration
  dvl_scale: true
  # Max clone storage for DVL
  dvl_clone: 1 

  # # enable Camera exterisic rotation calibration 
  # cam_exterisic_R: true
  # # enable Camera exterisic translation calibration 
  # cam_exterisic_p: true
  # # enable Camera time offset calibration
  # cam_timeoffset: true
  # # Max clone storage for Camera
  # cam_clone: 9